pkgs/applications/networking/instant-messengers/element/element-desktop.nix

```diff
@@ -123,6 +123,10 @@ stdenv.mkDerivation (finalAttrs: builtins.removeAttrs pinData [ "hashes" ] // {
     mimeTypes = [ "x-scheme-handler/element" ];
   };
 
+  postFixup = lib.optional stdenv.isDarwin ''
```

Comment:
```suggestion
  postFixup = lib.optionalString stdenv.isDarwin ''
```

Reply:
This doesn't seem to have a functional difference, but applied anyway.

Reply:
`optional` outputs a list: https://noogle.dev/?selected=%22lib.lists.optional%22&term=%22optional%22

I think it works because if `postFixup` is a list, the items of the list get concatenated together (probably with a `\n` character).